nixos/modules/services/misc/gitea.nix

```diff
@@ -569,7 +569,10 @@ in
         Restart = "always";
         # Runtime directory and mode
         RuntimeDirectory = "gitea";
-        RuntimeDirectoryMode = "0755";
+        RuntimeDirectoryMode = "0750";
```

Comment:
FYI when connecting to gitea via a unix socket you must add nginx to the gitea group if not already done

Reply:
There are no options in the `gitea` service to work with nginx, so for there is only a manual variant:
```
  users.users.nginx = {
    extraGroups = [ "gitea" ];
  };
```
Someday I'll have to do that. If I don't forget.